### PR TITLE
Feature/adding css

### PR DESCRIPTION
**Pull request by Dakova1994**
Add CSSs for our current UI

### CODE REVIEW

File: Projekt/src/app/app.component.html [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
-    <h2><a target="_blank" rel="noopener" href="https://angular.io/tutorial">Tour of Heroes</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://github.com/angular/angular-cli/wiki">CLI Documentation</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-  </li>
-</ul>
+<div class="conteiner">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: `conteiner` -> `container`

File: Projekt/src/app/app.component.html [hunk 2]
```diff
@@ -1,20 +1,34 @@
...
 
+  <div class="menuForm">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: It's not a form. `menuForm` -> `menu`

File: Projekt/src/app/app.component.html [hunk 3]
```diff
@@ -1,20 +1,34 @@
...
+    Home Profile
+  </div>
+
+  <div class="halfHomeForm">
+
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: remove new line

File: Projekt/src/app/app.component.html [hunk 4]
```diff
@@ -1,20 +1,34 @@
...
-    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-  </li>
-</ul>
+<div class="conteiner">
 
+  <div class="menuForm">
+    Home Profile
+  </div>
+
+  <div class="halfHomeForm">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: `halfHomeForm` -> `panel`

File: Projekt/src/app/app.component.html [hunk 5]
```diff
@@ -1,20 +1,34 @@
...
+
+    Panel
+    <div class="panelForm">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: `panelForm` -> `panel__elements`

File: Projekt/src/app/app.component.html [hunk 6]
```diff
@@ -1,20 +1,34 @@
...
+
+      Cards to draw:
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: put this text to `<span>` tags

File: Projekt/src/app/app.component.html [hunk 7]
```diff
@@ -1,20 +1,34 @@
...
+      <div class="inputDrawForm">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: `inputDrawForm` -> `panel__cards_count`

Comment (emosGambler) [inline Projekt/src/app/app.component.html]: remove this class name. Input will have a class name

File: Projekt/src/app/app.component.html [hunk 8]
```diff
@@ -1,20 +1,34 @@
...
+        <input type="number" (input)="numberOfCards = $event.target.value" min="1" max="deck.remaining">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: add class name here `panel__input__cards__count`

File: Projekt/src/app/app.component.html [hunk 9]
```diff
@@ -1,20 +1,34 @@
...
+      </div>
+
+      <div class="drawButton">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: remove this class name

File: Projekt/src/app/app.component.html [hunk 10]
```diff
@@ -1,20 +1,34 @@
...
+        <button (click)="drawCards()">Draw!</button>
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: `class="panel__button__draw__cards"`

File: Projekt/src/app/app.component.html [hunk 11]
```diff
@@ -1,20 +1,34 @@
...
+      </div>
+
+    </div>
+  </div>
+  <div class="halfHomeForm">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: `halfHomeForm` -> `panel`

File: Projekt/src/app/app.component.html [hunk 12]
```diff
@@ -1,20 +1,34 @@
...
+    Info
+
+    <div class="showCards">
+      <div *ngIf="deck && pile">
+          <div *ngFor="let card of drawnCards">
+            <img class="card-photo" src="{{card.image}}">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: `card-photo` -> `panel__card__photo`

File: Projekt/src/app/app.component.html [hunk 13]
```diff
@@ -1,20 +1,34 @@
...
+      <div class="drawButton">
+        <button (click)="drawCards()">Draw!</button>
+      </div>
+
+    </div>
+  </div>
+  <div class="halfHomeForm">
+    Info
+
+    <div class="showCards">
```
Comment (emosGambler) [inline Projekt/src/app/app.component.html]: unused class name. please remove

File: Projekt/src/app/app.component.ts [hunk 14]
```diff
@@ -1,10 +1,32 @@
...
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
-  title = 'app';
+export class AppComponent implements OnInit {
+    public title = 'app';
+    public pile: IPile;
```
Comment (emosGambler) [inline Projekt/src/app/app.component.ts]: `pile` -> `currentPile`

File: Projekt/src/app/app.component.ts [hunk 15]
```diff
@@ -1,10 +1,32 @@
...
+    public deck: IDeck;
```
Comment (emosGambler) [inline Projekt/src/app/app.component.ts]: `deck` -> `currentDeck`

File: Projekt/src/app/app.component.ts [hunk 16]
```diff
@@ -1,10 +1,32 @@
...
+    public drawnCards = [];
+    public myCards = [];
```
Comment (emosGambler) [inline Projekt/src/app/app.component.ts]: is it used somewhere? If not, please remove

File: Projekt/src/app/app.component.ts [hunk 17]
```diff
@@ -1,10 +1,32 @@
...
+        this.cardsService.getShuffledDeck().subscribe( shuffledDeck  => {
+            this.deck = shuffledDeck ;
+        });
+    }
+    drawCards() {
+        this.cardsService.getCards(this.deck.deck_id, this.numberOfCards).subscribe( pile  => {
+            this.pile = pile ;
+            this.pile.cards.map(card => {
+                this.drawnCards.push(card);
+                console.log()
```
Comment (emosGambler) [inline Projekt/src/app/app.component.ts]: remove this empty console log

**Comment by Dakova1994** [comment]
I fix everything i can.

**Pull request merged**